libs/service-portal/assets/src/screens/VehicleBulkMileage/VehicleBulkMileageRow.tsx

```diff
@@ -49,25 +53,25 @@ export const VehicleBulkMileageRow = ({ vehicle }: Props) => {
   })
 
   const [putAction] = usePutSingleVehicleMileageMutation({
-    onError: () => {
-      setPostError(formatMessage(m.errorTitle))
-      setPostStatus('error')
-    },
-    onCompleted: () => {
-      setPostError(null)
-      setPostStatus('put-success')
-    },
+    onError: () => handleMutationResponse(true),
+    onCompleted: ({ vehicleMileagePutV2: data }) =>
+      handleMutationResponse(
+        data?.__typename === 'VehiclesMileageUpdateError',
+        data?.__typename === 'VehiclesMileageUpdateError'
+          ? data?.message ?? formatMessage(m.errorTitle)
+          : undefined,
+      ),
   })
 
   const [postAction] = usePostSingleVehicleMileageMutation({
-    onError: () => {
-      setPostError(formatMessage(m.errorTitle))
-      setPostStatus('error')
-    },
-    onCompleted: () => {
-      setPostError(null)
-      setPostStatus('post-success')
-    },
+    onError: () => handleMutationResponse(true),
+    onCompleted: ({ vehicleMileagePostV2: data }) =>
+      handleMutationResponse(
+        data?.__typename === 'VehiclesMileageUpdateError',
```

Comment:
Can this not be typed from the schema?

Reply:
it's a conditional graphql typed response, must typename so eslint knows which one